scout/server/utils.py

```diff
@@ -118,27 +118,36 @@ def case_append_alignments(case_obj):
     Args:
         case_obj(scout.models.Case)
     """
-    case_obj['bam_files'] = []
-    case_obj['mt_bams'] = []
-    case_obj['bai_files'] = []
-    case_obj['mt_bais'] = []
-    case_obj['sample_names'] = []
-
-    bam_files = [
-        ('bam_file','bam_files', 'bai_files'),
-        ('mt_bam', 'mt_bams', 'mt_bais')
-    ]
+    analysis = [{'path':'bam_file', 'append_to':'bam_files', 'index':'bai_files'},
+                {'path':'mt_bam', 'append_to':'mt_bams', 'index':'mt_bais'},
+                {'path':'rhocall_bed', 'append_to':'rhocall_beds', 'index':'no_index'},
+                {'path':'rhocall_wig', 'append_to':'rhocall_wigs', 'index':'no_index'},
+                {'path':'upd_regions_bed', 'append_to':'upd_regions_beds', 'index':'no_index'},
+                {'path':'upd_sites_bed', 'append_to':'upd_sites_beds', 'index':'no_index'},
+                {'path':'tiddit_coverage_wig', 'append_to':'tiddit_coverage_wigs', 'index':'no_index'}]
 
     for individual in case_obj['individuals']:
-        case_obj['sample_names'].append(individual.get('display_name'))
-        for bam in bam_files:
-            bam_path = individual.get(bam[0])
-            if not (bam_path and os.path.exists(bam_path)):
+        case_obj['sample_names'] =(individual.get('display_name'))
+        for a in analysis:
```

Comment:
Same here, a few more chars for the var name: I know the scope is limited, but just in case the loop contents grows longer. ☺️

Reply:
Old habits from other coding conventions  :)